lib/railway_ipc/command_message_handler.ex

```diff
@@ -4,13 +4,18 @@ defmodule RailwayIpc.CommandMessageHandler do
       :ok ->
         :ok
 
-        {:error, _error} = e -> e
+      {:error, _error} = e ->
+        e
 
       {:emit, event} ->
         {:emit, prepare_event(event, message)}
     end
   end
 
+  defp prepare_event(%RailwayIpc.Persistence.PublishedMessage{} = event, _message) do
```

Comment:
This is kind of hacky--b/c the `RepublishedMessagesConsumer` returns `{:emit, persisted_published_message` from `handle_in/1`, we need a version of `prepare_event` that does nothing. The original code flow is that any `handle_in/1` can return `{:emit, protobuf}`, but for message re-publishing flow, we don't want to return the protobuf. We need to return the persisted published message record b/c that is the entity that knows the exchange to which it needs to be re-published.